src/main/java/org/testeditor/fixture/web/WebDriverFixture.java

```diff
@@ -128,7 +138,8 @@ public void setExecuteScript(String exec) {
      * <li><b>chrome</b> - opens Google Chrome</li>
      * </ul>
      * 
-     * @param browser String literal for used browser
```

Comment:
please put those changes in a pretty print pull in the future, it distracts from reviewing this pull.